.circleci/config.yml

```diff
@@ -0,0 +1,118 @@
+            python3 initialize.py
+      - save_cache:
+          key: deps1-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ checksum ".circleci/ci-requirements.txt" }}
+          paths:
+            - "venv"
+  deploy: # Deploy code to main app on merge
+    docker: 
+      - image: cimg/python:3.6
+    steps:
+      - checkout
+      - run: sudo chown -R circleci:circleci /usr/local/bin
+      - restore_cache:
+          key: deps1-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ checksum ".circleci/ci-requirements.txt" }}
+      - run:
+          name: Install python dependencies in virtual environment
+          command: |
+            cd .circleci/
+            python3 -m venv venv
+            source venv/bin/activate
+            pip install --upgrade pip
+            pip install -r ci-requirements.txt
+      - run:
+          name: Run Deploy Command
+          command: |
+            cd .circleci/
+            env
+            source venv/bin/activate
+            python3 deploy.py
+      - save_cache:
+          key: deps1-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ checksum ".circleci/ci-requirements.txt" }}
+          paths:
+            - "venv"
+  delete: # Delete old review apps on schedule or on merge
+    docker: 
+      - image: cimg/python:3.6
+    steps:
+      - checkout
+      - restore_cache:
+          key: deps2-{{ .Branch }}-{{ checksum ".circleci/ci-requirements.txt" }}
+      - run:
+          name: Install python dependencies in a virtual environment
+          command: |
+            cd .circleci/
+            python3 -m venv venv
+            source venv/bin/activate
+            pip install --upgrade pip
+            pip install -r ci-requirements.txt
+      - run:
+          name: Run Delete Command 
+          command: |
+            echo "Deleting expired review apps and databases..."
+            cd .circleci/
+            env
+            source venv/bin/activate
+            python3 delete.py
+      - save_cache:
+          key: deps2-{{ .Branch }}-{{ checksum ".circleci/ci-requirements.txt" }}
+          paths:
+            - "venv"
+workflows:
+  deploy-workflow: # Initialize and deploy review app on commit
+    jobs:
+      - initialize
+      - deploy:
+          requires:
+            - initialize
+          filters:
+            branches:
+              ignore:
+                - main
+  merge-workflow: # Deploy review app to main app on merge, then delete review app
+    jobs:
+      - deploy:
+          filters:
+            branches:
+              only:
+                - main
+      - delete:
+          requires:
+            - deploy
+          filters:
+            branches:
+              only:
+                - main
+  schedule-workflow: # Delete review app on a schedule
+    triggers:
+      - schedule:
+          cron: "0 0 * * 1-5" # 00:00 from Monday to Friday
+          filters:
+            branches:
+              ignore:
+                - main
```

Comment:
I was under the impression that we wanted the delete script to _only_ run on `main`

Reply:
and in fact that's what we have above in `delete`. So it's not clear to me how this is working